### PR TITLE
make default output be input.html

### PR DESCRIPTION
**Pull request by maxandersen**
- add import of Nb for ease copy/pasting from examples
- default outputpath to be inputname.html

### CODE REVIEW

File: jnotebook-core/src/main/java/tech/catheu/jnotebook/Main.java [hunk 1]
```diff
@@ -109,7 +109,7 @@ public static class RenderConfiguration extends SharedConfiguration {
                             description = "The path to the notebook to render.")
     public String inputPath;
 
-    @CommandLine.Parameters(index = "1", description = "The output path.")
+    @CommandLine.Parameters(index = "1", arity = "0..1", description = "The output path.")
     public String outputPath;
```
Comment (cyrilou242) [inline jnotebook-core/src/main/java/tech/catheu/jnotebook/Main.java:113]: nit: add @Nullable annotation?

File: jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java [hunk 2]
```diff
@@ -13,6 +13,7 @@
  */
 package tech.catheu.jnotebook;
 
+import com.google.common.io.Files;
```
Comment (cyrilou242) [inline jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java:16]: nit: can't remember if it is an explicitly declared dependency
I'll check

File: jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java [hunk 3]
```diff
@@ -89,7 +90,11 @@ public void render(final Main.RenderConfiguration config) {
         html = optimizeHtml(html);
       }
 
-      final File outputFile = FileUtils.getFile(config.outputPath);
+      var outputPath = config.outputPath;
```
Comment (cyrilou242) [inline jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java:93]: nit: replace var by String

**Review by cyrilou242 — APPROVED** [review]
very nice thanks!
I think this should be the preferred way to generate html now.
The Getting Started doc should be updated.

I'll take care of the few nits later, feel free to comment if you think they are not relevant.

File: jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java [hunk 4]
```diff
@@ -89,7 +90,11 @@ public void render(final Main.RenderConfiguration config) {
...
+      if(outputPath==null) {
```
Comment (cyrilou242) [inline jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java:94]: nit for me: introduce 

LangUtils.java
```
public static Optional<T> optional(T o) {
  return Optional.ofNullable(o);
}
```
and use it.

File: jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java (hunk 2 above)
Comment (maxandersen) [reply jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java:16]: it was there before.

File: jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java (hunk 4 above)
Comment (maxandersen) [reply jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java:94]: not sure what you mean if outputPath is nullable the above is the simplest code?

Comment (cyrilou242) [reply jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java:94]: I was meaning: 
```
final String outputPath = optional(config.outputPath)
  .orElse(Files.getNameWithoutExtension(config.inputPath) + ".html");
```

but it's for me later no worries.

File: jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java [hunk 5]
```diff
@@ -89,6 +90,10 @@ public void render(final Main.RenderConfiguration config) {
         html = optimizeHtml(html);
       }
 
+      String outputPath = config.outputPath;
+      if(outputPath==null) {
+        outputPath = Files.getNameWithoutExtension(config.inputPath) + ".html";
+      } 
       final File outputFile = FileUtils.getFile(config.outputPath);
```
Comment (cyrilou242) [inline jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java:97]: `inputPath` should be used here right?

**Pull request merged**